### PR TITLE
README.md: Add about section

### PR DESCRIPTION
**Pull request by meetmangukiya**
Closes https://github.com/coala/corobo/issues/228

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,28 @@
 [![Codecov branch](https://img.shields.io/codecov/c/github/coala/corobo/master.svg)](https://codecov.io/gh/coala/corobo)
 [![Semaphore branch](https://semaphoreci.com/api/v1/coala/corobo/branches/master/badge.svg)](https://semaphoreci.com/coala/corobo)
 
+## About
+
+`corobo` is a python based chatbot that could be connected to many messaging
```
Comment (refeed) [inline README.md]: Uppercase p for Python?

File: README.md [hunk 2]
```diff
@@ -4,6 +4,28 @@
...
+platform including Slack, HipChat, Telegram, IRC, XMPP, Tox, Campfire, Gitter,
+Skype. Check all the backends available
+[here](http://errbot.io/en/latest/features.html#multiple-server-backends)
+
+This repository is mainly a collection of plugins that are written for
+[errbot](http://errbot.io). Errbot is a python based chatbot framework.
```
Comment (refeed) [inline README.md]: here as well

File: README.md [hunk 3]
```diff
@@ -4,6 +4,28 @@
...
+
+As described on Errbot website:
+> Errbot is a chatbot, a daemon that connects to your favorite chat service
+  and brings your tools into the conversation.
+
+It handles sending messages to correct rooms/person, receiving messages,
+calling your command routines on matching commands, etc. This command routines
+are added by writing plugins, which are collected and loaded by errbot when it
```
Comment (refeed) [inline README.md]: uppercase 'e', for Errbot

**Comment by meetmangukiya** [comment]
@refeed PTAL?

File: README.md [hunk 4]
```diff
@@ -4,6 +4,31 @@
...
+  and brings your tools into the conversation.
+
+It handles sending messages to correct rooms/person, receiving messages,
+calling your command routines on matching commands, etc. This command routines
+are added by writing plugins, which are collected and loaded by Errbot when it
+is ran.
+
+## Features
+
+Read more about what `corobo` do for you
```
Comment (refeed) [inline README.md]: does?

Comment (meetmangukiya) [reply README.md]: "could do for you" amended

File: README.md [hunk 5]
```diff
@@ -4,6 +4,31 @@
 [![Codecov branch](https://img.shields.io/codecov/c/github/coala/corobo/master.svg)](https://codecov.io/gh/coala/corobo)
 [![Semaphore branch](https://semaphoreci.com/api/v1/coala/corobo/branches/master/badge.svg)](https://semaphoreci.com/coala/corobo)
 [![Gitter Room](https://img.shields.io/badge/gitter-join%20chat%20%E2%86%92-brightgreen.svg)](https://gitter.im/coala/corobo)
+
+## About
+
+`corobo` is a Python based chatbot that could be connected to many messaging
+platform including Slack, HipChat, Telegram, IRC, XMPP, Tox, Campfire, Gitter,
```
Comment (refeed) [inline README.md]: `platforms` since it's in plural context

File: README.md [hunk 6]
```diff
@@ -4,6 +4,31 @@
...
+
+This repository is mainly a collection of plugins that are written for
+[errbot](http://errbot.io). Errbot is a Python based chatbot framework.
+
+As described on Errbot website:
+> Errbot is a chatbot, a daemon that connects to your favorite chat service
+  and brings your tools into the conversation.
+
+It handles sending messages to correct rooms/person, receiving messages,
+calling your command routines on matching commands, etc. This command routines
```
Comment (refeed) [inline README.md]: ...`These` command routines

**Comment by meetmangukiya** [comment]
@refeed PTAL

**Comment by meetmangukiya** [comment]
ack 23505873

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**